client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -14,57 +14,57 @@ varying vec3 eyeVec;
 varying vec3 lightVec;
 varying vec3 tsEyeVec;
 varying vec3 tsLightVec;
-varying float area_enable_parallax;
-varying float disp;
+varying float area_enable_parallax;
+varying float disp;

```

Comment:
Just noticed what have you done here last time you touched shaders.
We are using "varying" on variable when its common between vertex and pixel shader, to pass the values from one to another.
You shall define that one inside main() of the shader where its used.